Sources/SwiftProtobuf/TextFormatScanner.swift

```diff
@@ -216,6 +216,25 @@ internal struct TextFormatScanner {
         }
     }
 
+    private mutating func parseUTF8Identifier() -> UnsafeBufferPointer<UInt8>? {
```

Comment:
Why optional, seems like this will always return something, it might just have a count of zero?